### PR TITLE
Update site content

### PR DESCRIPTION
**Pull request by zapbot**
From:
zaproxy/zaproxy-website@981b320bfbc30256637eeb2e99e336c7e7fe43d7

### CODE REVIEW

File: search/index.json [hunk 1]
```diff
@@ -501,7 +501,7 @@
     "keywords": ["","default","directory","is","that","the","uses?","what","zap"],
     "tags": null,
```
Comment (kingthorin) [inline search/index.json:502]: I don't remember seeing this before, is that going to be an issue?

Comment (psiinon) [reply search/index.json:502]: Er, where did that come from??

Comment (kingthorin) [reply search/index.json:502]: <shrug> 🤷

Comment (thc202) [reply search/index.json:502]: That's normal.

**Review by thc202 — APPROVED** [review]

File: faq/what-is-the-default-directory-that-zap-uses/index.html [hunk 2]
```diff
@@ -146,8 +146,8 @@ <h2 id="linux">Linux</h2>
 <p><code>~/.ZAP</code></p>
 <h2 id="mac-os">Mac OS</h2>
 <p><code>~/Library/Application Support/ZAP</code></p>
-<h2 id="weekly-releases">Weekly releases</h2>
```
Comment (thc202) [inline faq/what-is-the-default-directory-that-zap-uses/index.html:149]: Not sure if we have links using this anchor, it will no longer work.

**Review by psiinon — APPROVED** [review]

File: faq/what-is-the-default-directory-that-zap-uses/index.html (hunk 2 above)
Comment (psiinon) [reply faq/what-is-the-default-directory-that-zap-uses/index.html:149]: Dont know if we have perminant ones, but I've def linked to it in posts in the past...

Comment (kingthorin) [reply faq/what-is-the-default-directory-that-zap-uses/index.html:149]: Should be okay, I think:
image

File: search/index.json (hunk 1 above)
Comment (kingthorin) [reply search/index.json:502]: Thanks!

File: faq/what-is-the-default-directory-that-zap-uses/index.html (hunk 2 above)
Comment (thc202) [reply faq/what-is-the-default-directory-that-zap-uses/index.html:149]: Was not referring specifically to the site (in any case the page is small so should be easy to see).

**Pull request merged**